site-src/geps/pending-geps.md

```diff
@@ -0,0 +1,58 @@
+
+|                     | GEP-1619: Session Persistence | GEP-1867: Per-Gateway Infrastructure | GEP-957: Destination Port Matching | GEP-1016: GRPCRoute | GEP-1324: Service Mesh in Gateway API | GEP-1426: xRoutes Mesh Binding | GEP-1651: Gateway Routability | GEP-1686: Mesh conformance testing plan | GEP-1709: Conformance Profiles | GEP-1742: HTTPRoute Timeouts | GEP-1748: Gateway API Interaction with Multi-Cluster Services | GEP-1897: BackendTLSPolicy | GEP-1911: Backend Protocol Selection | GEP-2162: Supported features in GatewayClass Status | GEP-2257: Gateway API Duration Format |
+|---------------------|-------------------------------|--------------------------------------|------------------------------------|---------------------|---------------------------------------|--------------------------------|-------------------------------|-----------------------------------------|--------------------------------|------------------------------|---------------------------------------------------------------|----------------------------|--------------------------------------|-----------------------------------------------------|---------------------------------------|
+| _Status_            | P                             | P                                    | E                                  | E                   | E                                     | E                              | E                             | E                                       | E                              | E                            | E                                                             | E                          | E                                    | E                                                   | E                                     |
+| _Compliance_        | NS                            | NS                                   | NS                                 |                     |                                       |                                |                               |                                         |                                |                              |                                                               | NS                         |                                      |                                                     |                                       |
+| **EKS**             |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
```

Comment:
Nit: I'd recommend sorting these alphabetically by the name as shown in this table.

Reply:
They were alphabetical by real name, but I had shortened them here.  I will add the real name.